File: GTTS/INL.SearchService/Models/SearchPersons_Result.cs
using System.Collections.Generic;

namespace INL.SearchService.Models
{
    public class SearchPersons_Result : ISearchPersons_Result
    {
        public List<ISearchPersons_Item> Collection { get; set; }
        public int Draw { get; set; }
        public int RecordsFiltered { get; set; }
        public int RecordsTotal { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.SearchService.Models/search-persons_result.ts


import { ISearchPersons_Item } from './isearch-persons_item';
import { ISearchPersons_Result } from './isearch-persons_result';

export class SearchPersons_Result implements ISearchPersons_Result {
  
	public Collection?: ISearchPersons_Item[];
	public Draw: number = 0;
	public RecordsFiltered: number = 0;
	public RecordsTotal: number = 0;
  
}


